Pipfile

```diff
@@ -19,6 +19,7 @@ scikit-learn = "*"
 pyside2 = "*"
 matplotlib = "*"
 cached-property = "*"
+pandas = "*"
```

Comment:
Why is this in this PR?

Reply:
unrelated, but required as the project depends on it. didn't feel like making a PR just for that.

Reply:
the project doesn't depend on it (at least it shouldn't), cached_property is standard lib